src/main/ui/app/route/About.jsx

```diff
@@ -6,18 +6,33 @@ export default class About extends Component {
     render() {
         return (
                 <div>
-                    <h3>About</h3>
+                    
+                    <h1>About</h1>
 
-                    <p>
-                        <Link to="/indexingStatus">Indexing Status</Link>
+                    <h3>1)How to add your module?</h3>
+
+                    <p> Adding your module to Add Ons for indexing is as easy as creating a pull request!
+                        Before we do that, we need to ensure that your module has been hosted on one of the sites that Add Ons currently supports indexing from: <br/>
+                        a) Bintray <br/>
+                        b) Modulus <br/>
```

Comment:
Instead of Modulus, call this "modules.openmrs.org".
(Technically Modulus is the name of the underlying software, not the tool we have running.)